test/test_dbcore.py

```diff
@@ -776,3 +777,24 @@ def test_no_results(self):
             self.db._fetch(ModelFixture1, dbcore.query.FalseQuery()).get()
             is None
         )
+
+
+class ExceptionTest(unittest.TestCase):
```

Comment:
Since we're migrating away from `unittest`, we've been writing all new tests using pytest. I reckon this one should be trivial to adapt?

Reply:
Done in https://github.com/beetbox/beets/pull/5383/commits/8283b4f78509c0d2464a40d327bad6121de5a5f6